src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,30 @@
+const ADD = 'books/ADD_BOOK';
+const REMOVE = 'books/REMOVE_BOOK';
+
+const INITIAL_STATE = [];
+
+const bookReducer = (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case ADD:
+      return [...state, action.book];
+
+    case REMOVE: {
+      const filteredBooks = state.filter((book) => book.id !== action.bookId);
+      return filteredBooks;
+    }
+    default:
+      return state;
+  }
+};
+
+export const addBook = (book) => ({
+  type: ADD,
+  book,
+});
+
+export const removeBook = (bookId) => ({
+  type: REMOVE,
+  bookId,
+});
```

Comment:
- [ ] You used the old way to create a redux slice (actions and reducer), You asked to use `redux toolkit` and that's not how to create a redux slice with redux-toolkit, Please fix that. See this [link](https://redux-toolkit.js.org/api/createslice) for more info.